app/screens/shared/resource-list/ResourceIcons.js

```diff
@@ -0,0 +1,49 @@
+import React, { PropTypes } from 'react';
+import Glyphicon from 'react-bootstrap/lib/Glyphicon';
+
+import { getHumanizedPeriod } from 'utils/resourceUtils';
+
+function renderIcon(glyph, text) {
+  if (!text) {
+    return null;
+  }
+
+  return (
+    <span>
+      <Glyphicon glyph={glyph} />
+      <span className="text">{text}</span>
+    </span>
+  );
+}
+
+function getHourlyPrice({ minPricePerHour, maxPricePerHour }) {
+  if (!(minPricePerHour || maxPricePerHour)) {
+    return null;
+  }
+  if (minPricePerHour && maxPricePerHour) {
+    return `${Number(minPricePerHour)} - ${Number(maxPricePerHour)}e/h`;
```

Comment:
- [ ] You could also add a check for a case where `minPricePerHour` and `maxPricePerHour` are equal for example both '10.00' so we avoid displaying '10 - 10e/h'. In those cases only show '10e/h'